src/mock/persondata/personDeltBosted.ts

```diff
@@ -120,18 +178,16 @@ export const personDeltBosted: Person = {
             sivilstandRelasjon: null
         }
     ],
-    foreldreansvar: [
-        {
-            ansvar: 'felles',
-            ansvarlig: null,
-            ansvarsubject: null
-        },
-        {
+    foreldreansvar: forelderBarnRelasjonMock
+        .filter(relasjon => relasjon.rolle !== ForelderBarnRelasjonRolle.BARN)
+        .map(forelder => ({
             ansvar: 'felles',
-            ansvarlig: null,
+            ansvarlig: {
+                navn: forelder.navn.firstOrNull(),
+                ident: forelder.ident
+            },
             ansvarsubject: null
-        }
-    ],
+        })),
```

Comment:
Vi lagde en del mockpersoner da vi testet med David, disse har blitt tungvindte å vedlikeholde nå i etterkant dersom vi gjør endringer i frontenden. Skal vi fjerne disse?

Reply:
trur det kan fjernes. Det som kan kikkes på er om dette kan legges inn som del av nav faker-oppsettet. For det å vedlikeholde eksakte verdier har vore vanskeleg tidlegare, og det å laga nav faker var ein billigare løsning enn vedlkeholdet av reelle verdier.